crates/oxc_transformer/src/common/statement_injector.rs

```diff
@@ -250,4 +255,12 @@ impl<'a> StatementInjectorStore<'a> {
     fn set_current_statement_address(&self, stmt: &Statement<'a>) {
         self.current_statement_address.set(stmt.address());
     }
+
+    // Assertion for checking if it still contains statements that don't inject
+    // `#[inline(always)]` because this is a no-op in release mode
+    #[expect(clippy::inline_always)]
+    #[inline(always)]
+    fn assert_insertions(&self) {
```

Comment:
I suggest giving this method a more descriptive name e.g. `assert_no_insertions_remaining`.

Reply:
Looks good to me!